cmd/tarocli/proofs.go

```diff
@@ -20,6 +20,7 @@ var proofCommands = []cli.Command{
 		Category:  "Proofs",
 		Subcommands: []cli.Command{
 			verifyProofCommand,
+			decodeProofCommand,
```

Comment:
So I think we can probably split up the diff into a few commits:
  * Add the new proto fields + run the code generation 
  * Implement the decode RPC
  * Add the new CLI command